script/verification/HoloceneSystemConfigUpgrade.s.sol

```diff
@@ -64,6 +64,7 @@ contract HoloceneSystemConfigUpgrade is SuperchainRegistry, VerificationBase {
     {
         systemConfigAddress = proxies.SystemConfig;
         sysCfg = ISystemConfig(proxies.SystemConfig);
+        addAllowedStorageAccess(systemConfigAddress); //TODO: check if necessary here
```

Comment:
@winnsterx I tried and this line seems to not be necessary could you explain me more why we add this here?